lib/souyuz/generators/build_command_generator.rb

```diff
@@ -44,7 +45,8 @@ def build_targets
       def targets
         targets = []
         targets += build_targets
-        targets << "-t:SignAndroidPackage" if Souyuz.project.android?
+        targets << "-t:SignAndroidPackage" if Souyuz.project.android? and !Souyuz.config[:keystore_path] and !Souyuz.config[:keystore_alias] and !Souyuz.config[:keystore_password]
```

Comment:
Could you group those conditions to a helper method for better readability, please?